packages/components/src/Spinner/Spinner.mdx

```diff
@@ -0,0 +1,53 @@
+---
+name: Spinner
+menu: Components
+route: /components/spinner
+---
+
+import { Playground, Props } from "docz";
+import { ComponentStatus } from "@jobber/docx";
+import { Spinner } from ".";
+import { Text } from "../Text";
+
+# Spinner
+
+<ComponentStatus stage="pre" responsive="no" accessible="yes" />
+
+Spinners are used to visualize an undetermined load time for a section of a page
+and prevent double clicking of a UI element by overlaying it while its action is
+processing.
+
+The default spinner should be used to indicate loading of a view
+
```

Comment:
@chris-at-jobber I updated the copy based on your comments left on #177, let me know if you have more thoughts here

Reply:
@whereyoubinns Thanks! I'm going to bring this down into the Design & Usage Guidelines with some minor tweaks to better reflect this version of Spinner.